### ISSUE TITLE
"hostsEntries" capability don't work if host name contains from one word etc. localhost

### ISSUE THREAD

**Issue by idulsky** [aerokube/selenoid#470]
I'm using selenoid.exe on Windows which is running one of the selenoid/vnc:xxx containers. I've noticed that when I'm tying to pass hostsEntries capablity from my test like 
```
var caps = new DesiredCapabilities("chrome", "67.0", new Platform(PlatformType.Any));
caps.SetCapability("hostsEntries", new [] { "localhost:192.168.174.225" });
```
container's browser cannot resolve "localhost". But when I'm passing like this
`caps.SetCapability("hostsEntries", new [] { "localhost.com:192.168.174.225" });`
hostname is resolved as expected. So tell me whether it's a Docker-specific issue or a Selenoid issue. And how can pass single-word domain via capabilities? Thank you.

**Comment by vania-pooh** [comment]
@idulsky a default `/etc/hosts` file already contains something like `localhost 127.0.0.1`. So I would not use `localhost` to not clash with Docker logic. E.g. `myhost:192.168.174.225` should work.

**Comment by idulsky** [comment]
Seems legit. But in out test environment we have a couple external services pointing to "localhost" and it's hardly changeable. Any workaround for "localhost" though?

**Comment by vania-pooh** [comment]
@idulsky probably you could try to completely override `/etc/hosts` by mounting a file as a volume. Selenoid allows to do this by specifying `volumes` property in `browsers.json`. See https://aerokube.com/selenoid/latest/#_other_optional_fields.

**Comment by idulsky** [comment]
When trying to override localhost entry in container's hosts file, after running I catch this error
```
[1530864673.722][INFO]: resolved localhost to ["192.168.174.225"]
[1530864674.723][DEBUG]: failed to connect to localhost (error -102)
[1530864674.775][INFO]: RESPONSE InitSession session not created exception from disconnected: unable to connect to renderer (Session info: chrome=67.0.3396.62)
[1530864674.775][DEBUG]: Log type 'driver' lost 0 entries on destruction
[1530864674.775][DEBUG]: Log type 'browser' lost 0 entries on destruction
```
Environment started as I see through VNC, but browser won't. I guess internal container logic depends on its own local host, which is now pointing on external (wrong) host. Chances are workaround isn't possible :(

**Comment by vania-pooh** [comment]
@idulsky yes, in that case no sure whether a workaround exists.
